.gitignore

```diff
@@ -38,3 +40,7 @@ python/kfserving/.travis.yml
 python/kfserving/git_push.sh
 python/kfserving/test-requirements.txt
 python/kfserving/tox.ini
+
+# Ignore files for MacOS
+.DS_Store
+config/.DS_Store
```

Comment:
FYI DS store will pop up in and folder on OSX that you browse to using Finder. I think `**/.DS_Store` will catch all of them: https://stackoverflow.com/questions/18393498/gitignore-all-the-ds-store-files-in-every-folder-and-subfolder

Reply:
Thanks, I updated the .gitignore.